flask_combo_jsonapi/data_layers/alchemy.py

```diff
@@ -596,7 +596,12 @@ def apply_nested_fields(self, data, obj):
 
                         nested_fields_to_apply.append({"field": key, "value": nested_objects})
                     else:
-                        nested_fields_to_apply.append({"field": key, "value": nested_model(**value)})
+                        nested_field = getattr(obj, key)
+                        if nested_field:
+                            for attribute,new_value in value.items():
+                                setattr(nested_field, attribute, new_value)
```

Comment:
Currently doesn't work with multiple nesting in the way that you can make partial update only first level of nesting. Its same as current solution where you replace whole instance only now you can have 1 level partial update.

Reply:
To work with multiple level nesting this will be little bit complicating to inspect class and if its relation ship check if its loaded. Currently multiple level nesting is also not load in `flask_combo_jsonapi.schema.get_nested_fields`. We can discuss how to improve this or just put in documentation that multiple nesting partial update is not supported (like it is now)

Reply:
Ok, let’s keep it this way for now